tests/__init__.py

```diff
@@ -33,24 +49,49 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         sys.stdout = self._original_stdout
 
 # ------------------- FIXTURES -------------------
+
+
 @pytest.fixture(autouse=True)
 def dj_config():
     if pathlib.Path('./dj_local_conf.json').exists():
         dj.config.load('./dj_local_conf.json')
     dj.config['safemode'] = False
+    dj.config["database.use_tls"] = False
     dj.config['custom'] = {
         'database.prefix': (os.environ.get('DATABASE_PREFIX') 
-                             or dj.config['custom']['database.prefix']),
+                            or dj.config['custom']['database.prefix']),
         'miniscope_root_data_dir': (os.environ.get('MINISCOPE_ROOT_DATA_DIR')
                                     or dj.config['custom']['miniscope_root_data_dir'])
     }
     return
 
 
+@pytest.fixture(autouse=True)
+def test_data(dj_config):
+    mini_root_dirs = dj.config['custom']['miniscope_root_data_dir']
+    
+    test_data_exists = np.all([find_full_path(mini_root_dirs, p).exists() 
+                               for p in sessions_dirs])
```

Comment:
```suggestion
    test_data_exists  = all(find_full_path(mini_root_dirs, p).exists() 
                               for p in sessions_dirs)
```

Reply:
when you convert to a numpy array, the entire generator is consumed, converted to a list, and then converted to np.array. Python's all stops at the first falsy element.